src/ui/MessageSearchItem/index.scss

```diff
@@ -71,7 +71,7 @@
   }
   .sendbird-message-search-item__left {
     @include themed() {
-      border-left: 4px solid t(primary-3);
+      border-inline-start: 4px solid t(primary-3);
```

Comment:
**as-is**
![Screenshot 2024-06-24 at 5 37 36 PM](https://github.com/sendbird/sendbird-uikit-react/assets/10060731/52d69b12-67e5-4a63-9314-de607d7f5156)
**to-be**
<img width="326" alt="Screenshot 2024-06-24 at 5 37 54 PM" src="https://github.com/sendbird/sendbird-uikit-react/assets/10060731/861ad639-cbbd-4aa3-a8d0-40c35475441e">